docs/guides/runtime-options-overview.mdx

```diff
@@ -53,6 +53,8 @@ Options can be defined before a primitive is constructed and passed to the primi
 <span id="options-table"></span>
 ### Available options
 
+The following table documents options from the latest version of `qiskit-ibm-runtime`. To see older option versions, visit the [`qiskit-ibm-runtime` API reference](/api/qiskit-ibm-runtime) and select a historical version.
```

Comment:
```suggestion
The following table documents options from the latest version of `qiskit-ibm-runtime`. To see older option versions, visit the [`qiskit-ibm-runtime` API reference](/api/qiskit-ibm-runtime) and select a previous version.
```
It would actually be "an historical", but some people find that awkward, so we can just change the word :)